src/Asset/Button/Button.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styles from '../CSS/Commonmodule.css';
+// import Images from '../Images/Images';
+
+const colors = ['#0088FE', '#00C49F', '#FFBB28'];
+const delay = 2500;
```

Comment:
Create common constant file in constants folder and export object containing single delay time property